### PR TITLE
platform: imx: Add 4 2048-byte blocks in memory map

### PR DESCRIPTION
**Pull request by paulstelian97**
Some drivers require allocating blocks larger than 1024 bytes. Since
i.MX has plenty of memory free for buffers, getting 8kB and turning them
into actual runtime memory blocks of size 2048 is fine.

Signed-off-by: Paul Olaru <paul.olaru@nxp.com>

### CODE REVIEW

**Comment by paulstelian97** [comment]
@tlauda Need second review.

File: src/platform/imx8/include/platform/lib/memory.h [hunk 1]
```diff
@@ -101,6 +101,7 @@
 #define HEAP_RT_COUNT256	32
 #define HEAP_RT_COUNT512	4
 #define HEAP_RT_COUNT1024	4
+#define HEAP_RT_COUNT2048	4
```
Comment (tlauda) [inline src/platform/imx8/include/platform/lib/memory.h]: You don't include it in the total size calculation on line 128.

Comment (paulstelian97) [reply src/platform/imx8/include/platform/lib/memory.h]: That only affected the debug (heap_show_...) or was it a deeper issue?

Comment (tlauda) [reply src/platform/imx8/include/platform/lib/memory.h]: Probably just debug.

Comment (tlauda) [reply src/platform/imx8/include/platform/lib/memory.h]: @paulstelian97 Actually it was more serious. You could potentially overwrite the next heap to the wrong base address calculated.

**Pull request merged**